### ISSUE TITLE
Different input resolution?

### ISSUE THREAD

**Issue by TommasoBendinelli** [j96w/DenseFusion#151]
Hi there, 
I managed to train the model correctly on my dataset and I pretty happy about results. 
I would like now to use as input an image with higher resolution. In what way should I change the network architecture?

**Comment by j96w** [comment]
Hi, a very interesting question. To be honest, I'm not sure which part of the network needs to be modified accordingly. Do you have some findings to share? For example, the efficiency problem after increased the resolution? Thanks.

**Comment by TommasoBendinelli** [comment]
Not really, I used the standard format 640x480. But it is definitely something to investigate. Especially for small pieces, better resolution might imply better performances.
